.github/workflows/plan-examples.py

```diff
@@ -10,14 +10,17 @@ def get_examples():
     """
     exclude = {
         'ai-ml/ray/terraform', # Skip until we fix CI script to test as per v5 convention
+        'ai-ml/ray/terraform/examples/pytorch', # Example relies on cluster created by parent blueprint
+        'ai-ml/ray/terraform/examples/xgboost', # Example relies on cluster created by parent blueprint
+        'ai-ml/jupyterhub', # Requires a domain name
         'analytics/terraform/datahub-on-eks/datahub-addon', # Internal module, covered under root example and not standalone
-        'analytics/terraform/datahub-on-eks/datahub-addon/modules/prereq' # Internal module, covered under root example and not standalone
+        'streaming/nifi', # Requires a domain name
     }
 
     projects = {
         x.replace('/versions.tf', '')
         for x in glob.glob('**/**/**/versions.tf', recursive=True)
-        if not re.match(r'^.+/_', x)
+        if not re.match(r'.*modules.*', x)
```

Comment:
updated filter to remove modules. modules on their own will not typically pass a plan since they require input from users. Since these are local modules that are used by the blueprints/examples here they are therefore tested via the blueprints that use them

Reply:
Agree. Plan should only run on Blueprints in the main folder. We will deprecate all local sub TF modules and use the upstream repos.